src/process/process.cr

```diff
@@ -112,6 +122,16 @@ class Process
     LibC.times(out tms)
     Tms.new(tms.utime / hertz, tms.stime / hertz, tms.cutime / hertz, tms.cstime / hertz)
   end
+
+  def self.daemonize(stdin : String = "/dev/null", stdout : String = "/dev/null", stderr : String = "/dev/null")
+    exit if fork
+    setsid
+    exit if fork
+    Dir.cd("/")
```

Comment:
This might result in unexpected behavior as the new daemonized process might attempt to read files relative to the directory it was started.

I couldn't find a reference of this being a good practice. Can you point us to the documentation about it?

Thank you.